packages/web-components/package.json

```diff
@@ -35,7 +35,7 @@
     "@open-wc/testing": "next",
     "@web/dev-server": "^0.1.22",
     "@web/test-runner": "^0.13.23",
-    "eslint": "^7.32.0",
+    "eslint": "^7.23.0",
```

Comment:
This is a downgrade, just in case that’s not intentional.

Reply:
Correct (nice catch). All packages use `7.23` and only one used `7.32`. Some linting errors popped up if switching all to `7.32` and I didn't want to introduce other changes here.